src/graphql/data/TopTokens.ts

```diff
@@ -59,7 +59,7 @@ export enum TokenSortMethod {
 
 export type PrefetchedTopToken = NonNullable<TopTokens100Query['response']['topTokens']>[number]
 
-function useSortedTokens(tokens: TopTokens100Query['response']['topTokens']) {
+function useSortedTokens(tokens: TopTokens100Query['response']['topTokens'] | undefined) {
```

Comment:
Why are you changing this type?

Reply:
cuz its now defaulting to undefined (see line 179) so we can make the check on line 182